blog/2025-02-21/index.md

```diff
@@ -0,0 +1,124 @@
+
+![Verified checkmark](verified.png)
+
+Over half of the apps on Flathub so far are verified, with the number regularly increasing.
+
+## App Store Clients
+
+Once an app is developed, submitted, tested, approved, built, and distributed, it appears in app store clients like Flathub.org, KDE Discover, GNOME Software, and elementary AppCenter—as well as the Flatpak CLI. While exact implementations vary and the presentation is up to the specific app store client, generally each will show:
+
+- Static permissions and their impact on safety
+- Open Age Rating Service rating and details
+- If an app uses outdated runtimes
+- Release notes for each release
+- If static permissions increase between releases
+
+Flathub.org and GNOME Software also display the app’s verified status.
+
+## Updates
+
+Once an app is accepted onto Flathub, it’s not just the wild west; there are still a number of safety protections built into the flow:
+
+- **Flathub maintains ownership over the manifest repo**, while app developers are invited as limited collaborators
+- **The manifest’s default branch is protected**, preventing direct pushes without a pull request
+- **The manifest’s commit history cannot be rewritten**, making it harder to sneak something in
+- **Flathub’s automated tests must pass** before a PR can be merged and an update can be pushed
+- **Static permission changes are held for human review** before an update is released to users
+- **Critical MetaInfo changes are held for human review**, e.g. if an app name, developer name, app summary, or license changes
+
+## Caveats
+
+There are a few caveats to some of the points above which I would be remiss not to mention. First, Flathub has granted a small handful of trusted partners (including Mozilla and OBS Studio) the ability to directly upload their builds from their own infrastructure. Even for the few cases of direct uploads, we require a public manifest and build pipeline to enable similar reproducibility and auditability as outlined above. We also require the apps to be verified, and still run automated tests such as our linter against them.
+
+Lastly, some apps (around 6%) use [extra-data](https://docs.flatpak.org/en/latest/conventions.html#exporting-through-extra-data) to instruct Flatpak to download and unpack an existing package (e.g. a Debian package) during installation. These are largely proprietary apps that cannot be built on Flathub’s infrastructure, or apps using complex toolchains that require network access during build. This is discouraged since it does not enable the same level of auditability nor multi-architecture support that building from source does. As a result, this is heavily scrutinized during human review and only accepted as a last resort.
```

Comment:
you should mention that extra-data apps during the "unpacking" runs in an tight Flatpak sandbox that does not allow host filesystem or network access. 

in technical terms this is equivalent to `--sandbox`. app developers cannot modify the permissions this extra-data sandbox.